simulator.py

```diff
@@ -166,17 +190,17 @@ def log_encounter(human1, human2, location, duration, distance, time):
                     'obs_lon': human2.obs_lon,
                 },
                 'human1': {
-                    'other_human_id': human1.name,
-                    'other_age': human1.age,
-                    'other_carefullness': human1.carefullness,
-                    'other_wearing_mask': human1.wearing_mask,
-                    'other_is_infected': human1.is_sick,
-                    'other_infection_timestamp': human1.infection_timestamp,
-                    'other_infectiousness': human1.infectiousness,
-                    'other_reported_symptoms': human1.reported_symptoms,
-                    'other_symptoms': human1.symptoms,
-                    'other_test_results': human1.test_results,
-                    'other_has_app': human1.has_app,
+                    'human_id': human1.name,
+                    'age': human1.age,
+                    'carefullness': human1.carefullness,
+                    'wearing_mask': human1.wearing_mask,
```

Comment:
Should this observation for human2 be structured in the similar way as human1 i.e. observed and unobserved?